src/constants/tooltips.js

```diff
@@ -35,5 +35,15 @@ export default {
     recoveryAddress: 'The address of the new wallet where you would like your recovered funds to be sent.',
     passphrase: 'The wallet passphrase of the migrated wallet.',
     twofa: 'Second factor authentication (2FA) code',
+  },
+  unsignedSweep: {
+    userKey: `Your user public key`,
```

Comment:
For consistency, might be nice to remind the user this is on the KeyCard.

Reply:
ok just updated